From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maciej=20=C5=BBenczykowski?= <maze@google.com>
Date: Wed, 10 Jun 2020 06:33:22 +0000
Subject: Revert "ANDROID: Remove default y on BRIDGE_IGMP_SNOOPING"

This reverts commit b4667bd57d67399b36167df1a1c8f9852c8095cf.

Reason for revert: Incorrect reasoning for change.

Change-Id: Iafbc2f3a07ca78a2f1421c6693d2c81c0085f3a8
---
 net/bridge/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/bridge/Kconfig b/net/bridge/Kconfig
--- a/net/bridge/Kconfig
+++ b/net/bridge/Kconfig
@@ -38,6 +38,7 @@ config BRIDGE_IGMP_SNOOPING
 	bool "IGMP/MLD snooping"
 	depends on BRIDGE
 	depends on INET
+	default y
 	---help---
 	  If you say Y here, then the Ethernet bridge will be able selectively
 	  forward multicast traffic based on IGMP/MLD traffic received from
